01-preface.md

```diff
@@ -40,8 +40,7 @@ to the following objective:
 \mathbb{E}\left[ \sum_{t=s}^T \beta^{t-s} \mathcal{U}_t | X_s\right]
```

Comment:
Do we really need the specific notation $\mathcal{U}_t$. It is a bit confusing as we later only use the instantaneous utility function denoted as $u_t$